userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -115,7 +146,9 @@ Kill Bill Administrator (or Kill Bill Admin):: The person responsible for instal
 
 [[Parent]]Parent:: An account that contains one or more child accounts. Setting up a parent-child associations lets you define which entity is responsible for paying the invoice. See also <<Child>>.
 
-Payment Method:: A record of payment details, such as for credit cards, debit cards, PayPal accounts, etc. Each account can have many payment methods and one default payment method. Note that much of the representation of a payment method is plugin-specific and is actually stored by the payment plugin itself.
+Payment Method:: A Kill Bill record that represents an abstraction corresponding to a payment scheme like a credit card, debit card, or PayPal. An account can have multiple payment methods corresponding to different payment gateways. For example, an account can have a credit card payment method as well as a Paypal payment method.Each account also has a *default* payment method associated with it. When the account needs to be charged for recurring payments, the default payment method is used. 
+
+Payment::  A Kill Bill record that encapsulates charging an end-user against a payment method. It has a state that indicates the status of the underlying transaction.
```

Comment:
I think what you have for `Payment` is more related to a `Payment Transaction`, so we could first define this, and then define `Payment` as a collection of `Payment Transaction` (e.g `authorization`, `refund`, ..) using the same `Payment Method`.

Reply:
Added `Payment Transaction`, `Payment Attempt` definitions and updated `Payment` definition based on what is specified in the [Payment slate docs](https://killbill.github.io/slate/#payment).

Reply:
It all looks good.